src/Validation/Validator.php

```diff
@@ -422,9 +422,16 @@ public function remove($field, $rule = null)
     }
 
     /**
-     * Sets whether a field is required to be present in data array.
+     * Sets whether a field is required to be present in data array. You can also pass array.
+     * Using an array will let you provide the following keys:
      *
-     * @param string $field the name of the field
+     * - `mode` individual mode for field
+     * - `message` individual error message for field
+     *
+     * You can also set mode and message for all passed fields, the individual settings
+     * takes precedence over group setting.
```

Comment:
i wanted 
'the individual setting takes precedence over group settings.' 
does this make more sense / is grammatically correct?